### PR TITLE
Cepetr/prodtest fix

### LINKED ISSUE #3325: Light up the display in prodtest
https://satoshilabs.slack.com/archives/C05RGL2BMPD/p1696406189928229

### LINKED ISSUE #3328: Document prodtest commands
add a `core/embed/prodtest/README.md` documenting all supported prodtest commands

### PR DESCRIPTION
**Pull request by cepetr**
This pull request contains two changes:

1) It's make the display (on T2B1 model) all white  when the prodtest is started 
2) It incorporates new  documentation to the prodtest protocol, including all supported command.

resolves #3325
resolves #3328

### CODE REVIEW

**Comment by matejcik** [comment]
please list out the Optiga commands too, just put "TBD" for text

**Comment by cepetr** [comment]
I've added TOUCH, BUTTON and SENS commands description as well as stubs for all Optiga related commands.

File: core/embed/prodtest/README.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+The command required two input parameters:
+* The first parameter specifies the expected button or combination of buttons, with possible values: LEFT, RIGHT, BOTH.
+* The seconds parameter specifies the timeout duration in seconds in range 1 to 9
+
+If the specified button or button combination is not detected within the timeout, the command will return and `ERROR TIMEOUT`.
+
+Example (to wait for 9 seconds for the left button)
+```
+TOUCH LEFT 9
```
Comment (matejcik) [inline core/embed/prodtest/README.md:86]:
```suggestion
BUTTON LEFT 9
```

**Review by matejcik — APPROVED** [review]
please autosquash & merge if CI passes

**Pull request merged**